src/javascript/app_2/Modules/Page404/Components/Icon404.jsx

```diff
@@ -0,0 +1,24 @@
+import classNames from 'classnames';
```

Comment:
@zakibinary 
According to PR #4577, we can import SVG files directly and use like other react component so we just need to save SVG files in `images` and import them like [the logo](https://github.com/binary-com/binary-static/pull/4577/files#diff-692734e21cd9894d9bcff2010d0d73a2R3)

P.S it's not a good idea to import all SVG files in JSX files because it makes the bundle file massive.

Reply:
@morteza-binary , sure will switch to adding SVG via import instead.

Reply:
@morteza-binary , after consideration of time and effort, I think for now we can keep the JSX implementation for the icon. 

Reason for this is that we apply a couple of global classes for modifying the `fill` and `stroke` to the paths in the SVG icon to suit `dark` and `light` theme. 

When using SVG Import, the `svgo` utility will remove and trim all of the classes inside the svg file which will remove the theme classes appended in the SVG icon. 

I have already optimized the SVG before appending it to the JSX file so in this case `svgo` isn't necessary.
 
I agree that it's not a good idea to import all SVG files into JSX, but for use-cases like this, where the icons are simple vector shapes and have maximum of two colors, I think we can keep the current implementation for now until we find a better way to manage/refactor theming.

Reply:
@morteza-binary we can postpone it after receiving the new design because we have to refactor the SCSS files too.